assets/tests/components/propertiesPanel/minischedule.test.tsx

```diff
@@ -214,6 +214,57 @@ const vehicle: Vehicle = {
   crowding: null,
 }
 
+const vehicleWithOffset: Vehicle = {
```

Comment:
Note that if this is mostly the same as `vehicle` you can use the spread operator to write this more concisely, something like `const vehicleWithOffset = { ...vehicle, overloadOffset: 8 }`.

Reply:
Yeah, I made this change in the next commit.